jquery.canvaswrapper.js

```diff
@@ -166,7 +166,11 @@
                                 }
 
                                 if (positions.length == 0) {
-                                    delete styleCache[key];
+                                    if (styleCache[key].measured) {
+                                        styleCache[key].measured = false;
+                                    } else {
+                                        delete styleCache[key];
```

Comment:
The thing is that some labels are "measured" (meaning the width and the height are read), but not placed since are off the bounds of the axis. They will be removed in the render pass, and then when they are needed again the next frame they aren't available.

This change delays the removal in the case the info is actually read but text not placed.